cogs/command_error.py

```diff
@@ -67,18 +67,22 @@ async def on_application_command_error(self, ctx: TeXBotApplicationContext, erro
             logging_message=logging_message,
         )
 
-        if isinstance(error, discord.ApplicationCommandInvokeError) and isinstance(error.original, GuildDoesNotExistError):  # noqa: E501
+        if (
+            isinstance(error, discord.ApplicationCommandInvokeError)
+            and isinstance(error.original, GuildDoesNotExistError)
+        ):
```

Comment:
Separate overly long selection statements into new variables rather than splitting across lines. Splitting line is less readable IMHO.

Fix:
```python
ERROR_IS_GUILD_DOES_NOT_EXIST: Final[bool] = bool(
    isinstance(error, discord.ApplicationCommandInvokeError)
    and isinstance(error.original, GuildDoesNotExistError)  # noqa: COM812
)
if ERROR_IS_GUILD_DOES_NOT_EXIST:
    ...
```